pandas/core/base.py

```diff
@@ -393,7 +406,16 @@ def _agg(arg, func):
                 """
                 result = {}
                 for fname, agg_how in arg.items():
-                    result[fname] = func(fname, agg_how)
+                    # OWO CHANGES
+                    import json
+                    try:
+                        items = json.loads(fname)
+                        _obj = {}
+                        for item in items:
+                            _obj[item] = self._gotitem(item, ndim=1, subset=None)
```

Comment:
This is kinda bad... but hopefully with a formal PR, we can get feedback on how to do this better